magento/pwa-studio
# [doc] Update links to old repo

## Description

Search and replace `magento-research` with `magento` in the project.

## Related Issue

Closes #1507 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Build site: `npm run develop`
3. Verify site builds
4. Go to a page that had a link to the old repo, such as `/peregrine`
5. Verify link goes to the new repo location
